### ISSUE TITLE
安装出现问题

### ISSUE THREAD

**Issue by 97classmate** [wifiphisher/wifiphisher#885]
[*] Starting Wifiphisher 1.4 ( https://wifiphisher.org ) at 2018-03-18 03:10
No handlers could be found for logger "wifiphisher.pywifiphisher"
[!] Failed to find an interface with monitor mode
[+] Captured credentials:
[!] Closing
### ``

**Comment by 97classmate** [comment]
How to solve it? Thank you

**Comment by JetseVerschuren** [comment]
How do you run wifiphisher?
Do you run `python bin/wifiphisher`?

**Comment by 97classmate** [comment]
Yes, I'm running like this, but it still doesn't work.

**Comment by anakin1028** [comment]
This is may be a issue of wifiphisher. Can you supply with `--logging` and see if program works for you?

**Comment by 97classmate** [comment]
`root@kali:/wifi/wifiphisher-1.4# wifiphisher --logging
[*] Starting Wifiphisher 1.4 ( https://wifiphisher.org ) at 2018-03-18 17:13
[!] Failed to find an interface with monitor mode
[+] Captured credentials:
[!] Closing
`

**Comment by 97classmate** [comment]
I tried other systems will not work

**Comment by anakin1028** [comment]
Ok from the new log, you have another issue is that your adaptor doesn't support `Monitor mode`.

**Comment by anakin1028** [comment]
You can use command `python bin/wifiphisher-e haha --logging`

**Comment by 97classmate** [comment]
I just tried according to your method, or not

**Comment by anakin1028** [comment]
Still show `cannot find monitor mode`?

**Comment by 97classmate** [comment]
`root@kali:/wifi/wifiphisher-1.4# python bin/wifiphisher -e haha --logging
[*] Starting Wifiphisher 1.4 ( https://wifiphisher.org ) at 2018-03-18 17:27
[!] Failed to find an interface with monitor mode
[+] Captured credentials:
[!] Closing
`   Nothing no shown

**Comment by 97classmate** [comment]
Nothing is shown

**Comment by anakin1028** [comment]
Ok I you need to supply one more argument `-nE`

**Comment by 97classmate** [comment]
Good support for you

**Comment by anakin1028** [comment]
We will take a look for your original issue. Have fun with the tool:)

**Comment by 97classmate** [comment]
OK thank you

**Comment by anakin1028** [comment]
When you run `python bin/wifiphisher` do you see `no handlers for the logging module`?

**Comment by anakin1028** [comment]
Since we have added the null handler for the logging module on the top level. Normally the tool should not output this message.

**Comment by 97classmate** [comment]
I don't know what's going on. The message I sent above will appear.

**Comment by 97classmate** [comment]
The key is this one     `Failed to find an interface with monitor mode
`

**Comment by anakin1028** [comment]
Yes. I'm just wondering why the logging handlers message is shown.

**Comment by 97classmate** [comment]
I don’t know if I’ve changed three systems today. I still have the same problem.
